### PR TITLE
Cube root function implementation and unit tests

### PR DESCRIPTION
**Pull request by Chalyan**

### CODE REVIEW

File: Building_abstractions_with_procedures/The_elements_of_programming/Procedures_as_Black_Box_Abstractions/cube_root_with_lexical_scoping/function_module.scm [hunk 1]
```diff
@@ -0,0 +1,9 @@
+(module functions (square cube)
+    (import scheme)
+    
+    (define (square x)
+        (* x x))
+    
+    (define (cube x)
```
Comment (amuradyan) [inline Building_abstractions_with_procedures/The_elements_of_programming/Procedures_as_Black_Box_Abstractions/cube_root_with_lexical_scoping/function_module.scm:7]: Please, add tests for this method.

File: Building_abstractions_with_procedures/The_elements_of_programming/Procedures_as_Black_Box_Abstractions/cube_root_with_lexical_scoping/function_module.scm [hunk 2]
```diff
@@ -0,0 +1,9 @@
+(module functions (square cube)
+    (import scheme)
+    
+    (define (square x)
```
Comment (amuradyan) [inline Building_abstractions_with_procedures/The_elements_of_programming/Procedures_as_Black_Box_Abstractions/cube_root_with_lexical_scoping/function_module.scm:4]: Please, add tests for `square` method.

**Review by amuradyan — CHANGES_REQUESTED** [review]
Awesome!!!!

File: Building_abstractions_with_procedures/The_elements_of_programming/Procedures_as_Black_Box_Abstractions/cube_root_with_lexical_scoping/function_module.scm (hunk 2 above)
Comment (Chalyan) [reply Building_abstractions_with_procedures/The_elements_of_programming/Procedures_as_Black_Box_Abstractions/cube_root_with_lexical_scoping/function_module.scm]: Added all needed tests for both 'square' and 'cube' functions. Please recheck.

**Comment by amuradyan** [comment]
Please, update the PR title, to make it more descriptive.

**Review by amuradyan — APPROVED** [review]
lgtm

**Pull request merged**